### ISSUE TITLE
Environment HOME path dialog is broken under windows

### ISSUE THREAD

**Issue by alexxn** [gitextensions/gitextensions#4690]
Probably due to slightly incresed DPI settings, on my Windows machine the dialog looks like this:

![image](https://user-images.githubusercontent.com/5682814/37771262-230ef9e8-2de8-11e8-9130-6e5e18fa5d34.png)

OK and Cancel buttons are missing.
Git Extensions v 2.51

**Comment by drewnoakes** [comment]
I see a similar situation on my laptop:

![image](https://user-images.githubusercontent.com/350947/37773488-a5532950-2dd5-11e8-9b29-5d1d583b4d6d.png)

Ideally the controls should be anchored. The form could be made resizable too, with a sensible minimum size.

**Comment by drewnoakes** [comment]
I spent some time looking into this and wasn't able to solve the problem, but did gather some information.

@alexnn, does this happen only if you change your display settings after rebooting? Are you on Windows 10?

My setup is high-DPI laptop screen, and 96dpi monitor.

- If I boot using only the high-DPI screen, GE looks fine
- If I boot using only the 96 DPI monitor, GE looks fine

Problems arise only when I boot on the high-DPI screen, then attach the 96 DPI monitor. In this case, lots of things look strange across many programs. Text and images render poorly in many applications (VS, Git Bash, ...). Some apps look fine though (Firefox, notepad.exe, ...). I don't know what the pattern is).

I've tried various combinations of settings in the WinForms designer for this screen (`FormFixHome.cs`) but couldn't find a combination that works reliably on all situations.

This problem applies across much of GE too, not just this dialog.

I hope to look further into DPI issues at some point in the future. I read a bit this afternoon, but will need to do more research to understand what the problem is.
